lib/MusicBrainz/Server/Controller/Account.pm

```diff
@@ -297,6 +297,13 @@ sub edit : Local RequireAuth DenyWhenReadonly {
             $form->value
         );
 
+        my $old_username = $editor->username;
```

Comment:
That should be `->name` (although this code doesn't actually run yet, due to an error below).